### PR TITLE
Update use cases to match implementation

### LINKED ISSUE #369: Update use cases on DG

### PR DESCRIPTION
**Pull request by DesSnowy**
Closes #369

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -265,22 +265,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-- 3a. KonTActs detects an error in the input format.
+- 1a. KonTActs detects an error in the input.
 
-  - 3a1.KonTActs requests for the corrected input.
-  - 3b2. User enters a new input.
-  - Steps 3a1 - 3a2 are repeated until input format is correct.
+  - 1a1.KonTActs rejects the addition request and shows an error message.
+  - 1b2. User enters a new input.
+  - Steps 1a1 - 3a1 are repeated until input entered is correct.
```
Comment (SwaminathanViswa) [inline docs/DeveloperGuide.md:279]: This should be '1a2' and not '3a1'

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -265,22 +265,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 **Extensions**
 
-- 3a. KonTActs detects an error in the input format.
+- 1a. KonTActs detects an error in the input.
 
-  - 3a1.KonTActs requests for the corrected input.
-  - 3b2. User enters a new input.
-  - Steps 3a1 - 3a2 are repeated until input format is correct.
+  - 1a1.KonTActs rejects the addition request and shows an error message.
+  - 1b2. User enters a new input.
```
Comment (SwaminathanViswa) [inline docs/DeveloperGuide.md:278]: This should be '1a2' and not '1b2'

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -322,22 +320,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-  - 4a1. KonTActs requests for the correct data.
-  - 4a2. User enters new data.
-  - Steps 4a1-4a2 are repeated until the data entered are correct.
+  - 1a1. KonTActs rejects the request and shows an error message.
+  - 1a2. User enters a new input.
+  - Steps 1a1-1a2 are repeated until the input entered is correct.
 
-    Use case resumes from step 4.
+    Use case resumes from step 1.
```
Comment (SwaminathanViswa) [inline docs/DeveloperGuide.md:336]: I feel this should be step 2 instead similar to line 308

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -507,30 +480,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+2. KonTActs imports the contact list from the specified file path and indicates successful import.
 
    Use case ends.
 
 **Extensions**
 
-- 2a. KonTActs detects an unsupported, corrupted file or a file without the valid headers.
+- 1a. KonTActs detects the file provided is invalid.
 
   - 2a1. KonTActs indicates it is unable to import from that file and gives a more specific error message.
```
Comment (SwaminathanViswa) [inline docs/DeveloperGuide.md:497]: This needs to be changed to '1a1' and '1a2'

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -507,30 +480,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
   - 2a2. User provides a new file path
   - Steps 2a1 - 2a2 are repeated until KonTActs is able to import contacts from the file.
 
     Use case resumes from step 3.
 
-- 2b. KonTActs detects invalid content in CSV file (e.g. Name/Email that does not comply with requirement, 
-Duplicate names)
+- 1b. KonTActs detects invalid content in CSV file.
 
     - 2b1. KonTActs indicates it is unable to import from that file and gives a more specific error message.
```
Comment (SwaminathanViswa) [inline docs/DeveloperGuide.md:505]: This needs to be changed to '2b1' to '2b2'

**Review by SwaminathanViswa — COMMENTED** [review]
LGTM! Just a bit of typos to fix!

**Review by Incogdino — APPROVED** [review]
LGTM I guess according to prof, specifying a specific file format is acceptable.

**Pull request merged**